internal/indexer/indexer.go

```diff
@@ -181,20 +181,21 @@ func (i *Indexer) handleEvent(evt event.Event) error {
 	cfg := config.GetConfig()
 	profileCfg := config.GetProfile()
 	logger := logging.GetLogger()
+	stg := storage.GetStorage()
```

Comment:
I know this is a bit of a bit, but can you change this to be `store` instead of `stg` as I think it'll be a little bit clearer to the future reader.